sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -90,83 +93,86 @@ export class ChangeFeedResponse<T> {
 
 // @public (undocumented)
 export class ClientContext {
-    constructor(cosmosClientOptions: CosmosClientOptions, globalEndpointManager: GlobalEndpointManager);
+    // Warning: (ae-forgotten-export) The symbol "ClientConfig" needs to be exported by the entry point index.d.ts
+    constructor(cosmosClientOptions: CosmosClientOptions, globalEndpointManager: GlobalEndpointManager, clientConfig: ClientConfig);
     // (undocumented)
-    batch<T>({ body, path, partitionKey, resourceId, options, diagnosticContext, }: {
+    batch<T>({ body, path, partitionKey, resourceId, options, diagnosticNode, }: {
```

Comment:
@v1k1 can you mention why are we adding diagnosticNode in every request?

Reply:
As discussed, `diagnosticNode` is populated in a dfs like execution of function executions. Hence it is added to function arguments.